corehq/apps/reports/static/reports/js/single_form.js

```diff
@@ -29,6 +29,8 @@ hqDefine("reports/js/single_form", function() {
             analyticsUsage('Edit Form Submission');
         });
 
+        hqImport("reports/js/readable_form").init();
```

Comment:
is it necessary to call init() even though it's called on page load in readable_form.js?

Reply:
Yeah, this code is for the case history tab in case data, where each row is a form and you click on the form to trigger an ajax request for content related to that form, including a readable version of the questions. So that content doesn't exist yet on page load.